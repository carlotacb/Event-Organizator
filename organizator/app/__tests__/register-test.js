import * as React from "react";
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from "react-test-renderer";
import RegisterPage from "../register";

it(`renders correctly`, () => {
  const tree = renderer.create(<RegisterPage />).toJSON();

  expect(tree).toMatchSnapshot();
});
