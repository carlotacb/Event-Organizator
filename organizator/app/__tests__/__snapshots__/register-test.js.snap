// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  style={
    [
      {
        "backgroundColor": "white",
        "paddingBottom": 50,
        "paddingLeft": 40,
        "paddingRight": 40,
        "paddingTop": 50,
      },
    ]
  }
>
  <RCTScrollView
    contentContainerStyle={
      {
        "paddingHorizontal": 20,
      }
    }
  >
    <View>
      <Text
        style={
          [
            {
              "color": "black",
              "fontSize": 40,
              "fontWeight": "bold",
              "textAlign": "center",
            },
          ]
        }
      >
        Welcome!
      </Text>
      <Text
        style={
          [
            {
              "color": "gray",
              "fontSize": 18,
              "marginBottom": 20,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 5,
              "textAlign": "center",
            },
          ]
        }
      >
        We are happy to have you here! Please fill in the following details
      </Text>
      <View
        style={
          [
            {
              "marginBottom": 20,
            },
          ]
        }
      >
        <View
          style={
            [
              {
                "display": "flex",
                "flexDirection": "row",
                "gap": 5,
              },
            ]
          }
        >
          <Text
            style={
              [
                {
                  "marginBottom": 8,
                  "marginLeft": 15,
                },
              ]
            }
          >
            Username
          </Text>
          <Text
            style={
              [
                {
                  "color": "red",
                },
              ]
            }
          >
            *
          </Text>
        </View>
        <View
          isFocus={false}
          style={
            [
              {
                "alignItems": "center",
                "backgroundColor": "#eaeaea",
                "borderColor": "#eaeaea",
                "borderRadius": 20,
                "borderWidth": 2,
                "display": "flex",
                "flexDirection": "row",
                "height": 55,
                "paddingBottom": 5,
                "paddingLeft": 20,
                "paddingRight": 20,
                "paddingTop": 5,
              },
            ]
          }
        >
          <Text />
          <TextInput
            iconName="user"
            label="Username"
            onBlur={[Function]}
            onChangeText={[Function]}
            onFocus={[Function]}
            required={true}
            style={
              [
                {
                  "color": "black",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "marginLeft": 10,
                  "marginRight": 10,
                },
                {
                  "outline": "none",
                },
              ]
            }
          />
        </View>
      </View>
      <View
        style={
          [
            {
              "marginBottom": 20,
            },
          ]
        }
      >
        <View
          style={
            [
              {
                "display": "flex",
                "flexDirection": "row",
                "gap": 5,
              },
            ]
          }
        >
          <Text
            style={
              [
                {
                  "marginBottom": 8,
                  "marginLeft": 15,
                },
              ]
            }
          >
            Email
          </Text>
          <Text
            style={
              [
                {
                  "color": "red",
                },
              ]
            }
          >
            *
          </Text>
        </View>
        <View
          isFocus={false}
          style={
            [
              {
                "alignItems": "center",
                "backgroundColor": "#eaeaea",
                "borderColor": "#eaeaea",
                "borderRadius": 20,
                "borderWidth": 2,
                "display": "flex",
                "flexDirection": "row",
                "height": 55,
                "paddingBottom": 5,
                "paddingLeft": 20,
                "paddingRight": 20,
                "paddingTop": 5,
              },
            ]
          }
        >
          <Text />
          <TextInput
            iconName="at"
            keyboardType="email-address"
            label="Email"
            onBlur={[Function]}
            onChangeText={[Function]}
            onFocus={[Function]}
            required={true}
            style={
              [
                {
                  "color": "black",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "marginLeft": 10,
                  "marginRight": 10,
                },
                {
                  "outline": "none",
                },
              ]
            }
          />
        </View>
      </View>
      <View
        style={
          [
            {
              "marginBottom": 20,
            },
          ]
        }
      >
        <View
          style={
            [
              {
                "display": "flex",
                "flexDirection": "row",
                "gap": 5,
              },
            ]
          }
        >
          <Text
            style={
              [
                {
                  "marginBottom": 8,
                  "marginLeft": 15,
                },
              ]
            }
          >
            Password
          </Text>
          <Text
            style={
              [
                {
                  "color": "red",
                },
              ]
            }
          >
            *
          </Text>
        </View>
        <View
          isFocus={false}
          style={
            [
              {
                "alignItems": "center",
                "backgroundColor": "#eaeaea",
                "borderColor": "#eaeaea",
                "borderRadius": 20,
                "borderWidth": 2,
                "display": "flex",
                "flexDirection": "row",
                "height": 55,
                "paddingBottom": 5,
                "paddingLeft": 20,
                "paddingRight": 20,
                "paddingTop": 5,
              },
            ]
          }
        >
          <Text />
          <TextInput
            iconName="lock"
            label="Password"
            onBlur={[Function]}
            onChangeText={[Function]}
            onFocus={[Function]}
            password={true}
            required={true}
            secureTextEntry={true}
            style={
              [
                {
                  "color": "black",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "marginLeft": 10,
                  "marginRight": 10,
                },
                {
                  "outline": "none",
                },
              ]
            }
          />
          <Text />
        </View>
      </View>
      <View
        style={
          [
            {
              "marginBottom": 20,
            },
          ]
        }
      >
        <View
          style={
            [
              {
                "display": "flex",
                "flexDirection": "row",
                "gap": 5,
              },
            ]
          }
        >
          <Text
            style={
              [
                {
                  "marginBottom": 8,
                  "marginLeft": 15,
                },
              ]
            }
          >
            Confirm your password
          </Text>
          <Text
            style={
              [
                {
                  "color": "red",
                },
              ]
            }
          >
            *
          </Text>
        </View>
        <View
          isFocus={false}
          style={
            [
              {
                "alignItems": "center",
                "backgroundColor": "#eaeaea",
                "borderColor": "#eaeaea",
                "borderRadius": 20,
                "borderWidth": 2,
                "display": "flex",
                "flexDirection": "row",
                "height": 55,
                "paddingBottom": 5,
                "paddingLeft": 20,
                "paddingRight": 20,
                "paddingTop": 5,
              },
            ]
          }
        >
          <Text />
          <TextInput
            iconName="lock"
            label="Confirm your password"
            onBlur={[Function]}
            onChangeText={[Function]}
            onFocus={[Function]}
            password={true}
            required={true}
            secureTextEntry={true}
            style={
              [
                {
                  "color": "black",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "marginLeft": 10,
                  "marginRight": 10,
                },
                {
                  "outline": "none",
                },
              ]
            }
          />
          <Text />
        </View>
      </View>
      <View
        style={
          [
            {
              "marginBottom": 20,
            },
          ]
        }
      >
        <View
          style={
            [
              {
                "display": "flex",
                "flexDirection": "row",
                "gap": 5,
              },
            ]
          }
        >
          <Text
            style={
              [
                {
                  "marginBottom": 8,
                  "marginLeft": 15,
                },
              ]
            }
          >
            First Name
          </Text>
          <Text
            style={
              [
                {
                  "color": "red",
                },
              ]
            }
          >
            *
          </Text>
        </View>
        <View
          isFocus={false}
          style={
            [
              {
                "alignItems": "center",
                "backgroundColor": "#eaeaea",
                "borderColor": "#eaeaea",
                "borderRadius": 20,
                "borderWidth": 2,
                "display": "flex",
                "flexDirection": "row",
                "height": 55,
                "paddingBottom": 5,
                "paddingLeft": 20,
                "paddingRight": 20,
                "paddingTop": 5,
              },
            ]
          }
        >
          <Text />
          <TextInput
            iconName="id-badge"
            label="First Name"
            onBlur={[Function]}
            onChangeText={[Function]}
            onFocus={[Function]}
            required={true}
            style={
              [
                {
                  "color": "black",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "marginLeft": 10,
                  "marginRight": 10,
                },
                {
                  "outline": "none",
                },
              ]
            }
          />
        </View>
      </View>
      <View
        style={
          [
            {
              "marginBottom": 20,
            },
          ]
        }
      >
        <View
          style={
            [
              {
                "display": "flex",
                "flexDirection": "row",
                "gap": 5,
              },
            ]
          }
        >
          <Text
            style={
              [
                {
                  "marginBottom": 8,
                  "marginLeft": 15,
                },
              ]
            }
          >
            Last Name
          </Text>
          <Text
            style={
              [
                {
                  "color": "red",
                },
              ]
            }
          >
            *
          </Text>
        </View>
        <View
          isFocus={false}
          style={
            [
              {
                "alignItems": "center",
                "backgroundColor": "#eaeaea",
                "borderColor": "#eaeaea",
                "borderRadius": 20,
                "borderWidth": 2,
                "display": "flex",
                "flexDirection": "row",
                "height": 55,
                "paddingBottom": 5,
                "paddingLeft": 20,
                "paddingRight": 20,
                "paddingTop": 5,
              },
            ]
          }
        >
          <Text />
          <TextInput
            iconName="id-badge"
            label="Last Name"
            onBlur={[Function]}
            onChangeText={[Function]}
            onFocus={[Function]}
            required={true}
            style={
              [
                {
                  "color": "black",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "marginLeft": 10,
                  "marginRight": 10,
                },
                {
                  "outline": "none",
                },
              ]
            }
          />
        </View>
      </View>
      <View
        style={
          [
            {
              "marginBottom": 20,
            },
          ]
        }
      >
        <View
          style={
            [
              {
                "display": "flex",
                "flexDirection": "row",
                "gap": 5,
              },
            ]
          }
        >
          <Text
            style={
              [
                {
                  "marginBottom": 8,
                  "marginLeft": 15,
                },
              ]
            }
          >
            Biography
          </Text>
          <Text
            style={
              [
                {
                  "color": "red",
                },
              ]
            }
          >
            *
          </Text>
        </View>
        <View
          isFocus={false}
          style={
            [
              {
                "alignItems": "center",
                "backgroundColor": "#eaeaea",
                "borderColor": "#eaeaea",
                "borderRadius": 20,
                "borderWidth": 2,
                "display": "flex",
                "flexDirection": "row",
                "height": 55,
                "paddingBottom": 5,
                "paddingLeft": 20,
                "paddingRight": 20,
                "paddingTop": 5,
              },
            ]
          }
        >
          <Text />
          <TextInput
            iconName="pencil"
            label="Biography"
            onBlur={[Function]}
            onChangeText={[Function]}
            onFocus={[Function]}
            required={true}
            style={
              [
                {
                  "color": "black",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "marginLeft": 10,
                  "marginRight": 10,
                },
                {
                  "outline": "none",
                },
              ]
            }
          />
        </View>
      </View>
      <View
        style={
          [
            {
              "alignItems": "center",
              "display": "flex",
              "marginTop": 15,
            },
          ]
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            [
              {
                "alignItems": "center",
                "backgroundColor": "darkslategray",
                "borderRadius": 20,
                "height": 55,
                "justifyContent": "center",
                "marginBottom": 10,
                "marginLeft": 10,
                "marginRight": 10,
                "marginTop": 10,
                "width": "60%",
              },
            ]
          }
        >
          <Text
            style={
              [
                {
                  "color": "white",
                  "fontSize": 18,
                  "fontWeight": "bold",
                },
              ]
            }
          >
            Register
          </Text>
        </View>
      </View>
    </View>
  </RCTScrollView>
</RCTSafeAreaView>
`;
